temporal/fx.go

```diff
@@ -55,6 +55,8 @@ import (
 	"go.temporal.io/server/common/headers"
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/log/tag"
+	"go.temporal.io/server/common/membership/ringpop"
+	"go.temporal.io/server/common/membership/static"
```

Comment:
this doesn't appear to be in the PR?

Reply:
Right; to make this PR simpler, I refactored it here first: https://github.com/temporalio/temporal/pull/5499